Sources/StreamChat/Utils/LazyCachedMapCollection.swift

```diff
@@ -70,6 +70,10 @@ public struct LazyCachedMapCollection<Element>: RandomAccessCollection {
             return value
         }
     }
+
+    public func append(_ element: Element) {
```

Comment:
Where do you use this one?

Reply:
It is not shown in the changes, but we are appending the Root of the thread in the `getReplies()` function at the end. Previously we were using the `Array`. Now we need for `LazyCachedMapCollection` to support this as well.